### PR TITLE
feat: add yunoimage rikky packages

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add the rikky' rikky module and all the plugins of yunoimage.

## Related Issue

#2 (Don't close)

## How Has This Been Tested?

Load the file with apm and verify the displayed text and installation.

## Checklist:

- [x] The XML file has been formatted by prettier
- [x] Updated the modification date in `mod.xml`.

### CODE REVIEW

File: data/packages_list.xml [hunk 1]
```diff
@@ -403,6 +403,110 @@
...
+		<pageURL>http://auls.client.jp/</pageURL>
+		<downloadURL>http://auls.client.jp/</downloadURL>
+		<latestVersion>2014/05/06</latestVersion>
+		<files>
+			<file archivePath="auls_confirmclose">plugins/auls_confirmclose.auf</file>
+		</files>
+	</package>
+
+	<package>
+		<id>aulsoutputpng</id>
```
Comment (hal-shu-sato) [inline data/packages_list.xml:453]: Is this for compatibility?

Comment (hal-shu-sato) [inline data/packages_list.xml:453]: Is this for compatibility?

**Review by hal-shu-sato — COMMENTED** [review]
I have one point to check.

File: data/packages_list.xml (hunk 1 above)
Comment (mitosagi) [reply data/packages_list.xml:453]: I just didn't notice that it should be capitalized. Either way, I'll leave it as is.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for adding them!

**Pull request merged**